### ISSUE TITLE
Cant show metrics with Prometheus Operator

### ISSUE THREAD

**Issue by lfdominguez** [lensapp/lens#2503]
**Describe the bug**
Can't see any metrics from cluster

**To Reproduce**
1. Go to 'Workloads/Pods'
2. Click on 'any pod'
3. See error

**Expected behavior**
That's show the pod's metrics

**Screenshots**
![image](https://user-images.githubusercontent.com/5604131/114283917-63418e80-9a1a-11eb-90bd-b79f958f5e09.png)
![image](https://user-images.githubusercontent.com/5604131/114283927-6e94ba00-9a1a-11eb-826b-60be5ca79466.png)
![image](https://user-images.githubusercontent.com/5604131/114283933-810ef380-9a1a-11eb-8d7f-fea244242db8.png)
![image](https://user-images.githubusercontent.com/5604131/114283950-adc30b00-9a1a-11eb-87f9-fbf46f9344e0.png)
![image](https://user-images.githubusercontent.com/5604131/114283957-b87da000-9a1a-11eb-9c9b-6ce467bc3a51.png)

**Environment (please complete the following information):**
- Lens Version: 4.2.1
- OS: GNU/Linux Manjaro
- Installation method (e.g. snap or AppImage in Linux): Arch AUR that's use the AppImage

**Logs:**
When you run the application executable from command line you will see some logging output. Please paste them here:
```json
{
  "error": {
    "name": "StatusCodeError",
    "statusCode": 400,
    "message": "400 - {\"status\":\"error\",\"errorType\":\"bad_data\",\"error\":\"invalid parameter 'start': cannot parse \\\"\\\" to a valid timestamp\"}",
    "error": {
      "status": "error",
      "errorType": "bad_data",
      "error": "invalid parameter 'start': cannot parse \"\" to a valid timestamp"
    },
    "options": {
      "timeout": 0,
      "resolveWithFullResponse": false,
      "json": true,
      "qs": {
        "query": "sum(kube_node_status_capacity{node=~\"k8s-1|k8s-2|k8s-3|k8s-4\", resource=\"pods\"})",
        "start": "1618082700",
        "end": "1618086300",
        "step": "60",
        "kubernetes_namespace": ""
      },
      "headers": {
        "Host": "3537383f-37ba-4706-adc4-b6acbd7d4d03.localhost:40663"
      },
      "uri": "http://localhost:40663/api-kube/api/v1/namespaces/monitoring/services/prometheus-k8s:9090/proxy/api/v1/query_range",
      "simple": true,
      "transform2xxOnly": false
    },
    "response": {
      "statusCode": 400,
      "body": {
        "status": "error",
        "errorType": "bad_data",
        "error": "invalid parameter 'start': cannot parse \"\" to a valid timestamp"
      },
      "headers": {
        "access-control-allow-origin": "*",
        "content-type": "application/json",
        "date": "Sat, 10 Apr 2021 20:25:58 GMT",
        "server": "nginx/1.18.0",
        "content-length": "117",
        "connection": "close"
      },
      "request": {
        "uri": {
          "protocol": "http:",
          "slashes": true,
          "auth": null,
          "host": "localhost:40663",
          "port": "40663",
          "hostname": "localhost",
          "hash": null,
          "search": "?query=sum%28kube_node_status_capacity%7Bnode%3D~%22k8s-1%7Ck8s-2%7Ck8s-3%7Ck8s-4%22%2C%20resource%3D%22pods%22%7D%29&start=1618082700&end=1618086300&step=60&kubernetes_namespace=",
          "query": "query=sum%28kube_node_status_capacity%7Bnode%3D~%22k8s-1%7Ck8s-2%7Ck8s-3%7Ck8s-4%22%2C%20resource%3D%22pods%22%7D%29&start=1618082700&end=1618086300&step=60&kubernetes_namespace=",
          "pathname": "/api-kube/api/v1/namespaces/monitoring/services/prometheus-k8s:9090/proxy/api/v1/query_range",
          "path": "/api-kube/api/v1/namespaces/monitoring/services/prometheus-k8s:9090/proxy/api/v1/query_range?query=sum%28kube_node_status_capacity%7Bnode%3D~%22k8s-1%7Ck8s-2%7Ck8s-3%7Ck8s-4%22%2C%20resource%3D%22pods%22%7D%29&start=1618082700&end=1618086300&step=60&kubernetes_namespace=",
          "href": "http://localhost:40663/api-kube/api/v1/namespaces/monitoring/services/prometheus-k8s:9090/proxy/api/v1/query_range?query=sum%28kube_node_status_capacity%7Bnode%3D~%22k8s-1%7Ck8s-2%7Ck8s-3%7Ck8s-4%22%2C%20resource%3D%22pods%22%7D%29&start=1618082700&end=1618086300&step=60&kubernetes_namespace="
        },
        "method": "GET",
        "headers": {
          "Host": "3537383f-37ba-4706-adc4-b6acbd7d4d03.localhost:40663",
          "accept": "application/json"
        }
      }
    }
  }
}
```

**Kubeconfig:**
Quite often the problems are caused by malformed kubeconfig which the application tries to load. Please share your kubeconfig, remember to remove any secret and sensitive information.
```
Not needed
```

**Additional context**
I think that is a prometheus api compatibility problem because the error is that lens query with a timestamp not recognized by prometheus service.

**Comment by jim-docker** [comment]
Is this the bitnami repo's Prometheus Operator release?

**Comment by lfdominguez** [comment]
this prometheus operator is installed from microk8s module....

**Comment by lfdominguez** [comment]
```
msg="Starting Prometheus" version="(version=2.20.0, branch=HEAD, revision=e5a06b483527d4fe0704b8fa3a2b475b661c526f)"
```

**Comment by VitaliAndrushkevich** [comment]
This can be helpfull
https://github.com/lensapp/lens/issues/180#issuecomment-862264308

**Comment by marcbachmann** [comment]
Fixes coming in #3653, using the default [`kube-prometheus-stack`](https://github.com/prometheus-community/helm-charts/tree/main/charts/kube-prometheus-stack) setup
